tests/test_utils.py

```diff
@@ -57,6 +37,11 @@ def test_stringlike():
     assert is_string_like(123) is False
 
 
```

Comment:
>>> from __future__ import unicode_literals
>>> '💩'
u'\U0001f4a9'

since we use unicode_literals, when somebody enters a unicode string in a search field, it will be saved in \uXXXX form. Without unicode_literals we get

>>> '💩'
'\xf0\x9f\x92\xa9'

which is already in the form we want it to be in.